JS/Lesson22/onder/app.jss

```diff
@@ -0,0 +1,13 @@
+let acc = document.getElementsByClassName("accordion");
+
+        for (let i = 0; i < acc.length; i++) {
+            acc[i].onclick = function () {

```

Comment:
The code here almost works. Right now, when a button clicked the first time, nothing happens due to the `if (panel.style.display === "block") {` check not getting the display property from the first attempt.
Since you also tried to make it work with css classes (a better approach then modifying css in js directly) I recommend to do like this:
app.js

```
let acc = document.getElementsByClassName("accordion");

for (let i = 0; i < acc.length; i++) {
    acc[i].onclick = function () {
        let panel = this.nextElementSibling;
        panel.classList.toggle("active");
    };
}
```

index.html
Add class `active` to **all panel divs** in HTML 
```
<div class="panel active">
```

CSS
```
.panel {
  display: none;
}

.panel.active { // make sure to select both panel and active classes to make sure that the css priority rules allows you to change the display property
  display: block;
}
```